containers.md

```diff
@@ -29,7 +29,7 @@ We have compiled a list of popular software within the container ecosystem that
 
 | Name                      | URL                           | Comment                |
 | :-----                    |:-----                         | :-----                 |
-| Istio	| https://github.com/istio/istio/releases/	| 1) [Arm64 support is now available via the master branch](https://github.com/istio/istio/issues/26652). To try it out follow: https://github.com/istio/istio/wiki/Dev-Builds#dev-release-information. Support will be included in the upcoming 1.15 release ETA ~1 month.<br>2) [Istio container build instructions](https://github.com/aws/aws-graviton-getting-started/blob/main/containers-workarounds.md#Istio)|
+| Istio	| https://github.com/istio/istio/releases/	| 1) [Arm64 support is now available via the master branch](https://github.com/istio/istio/issues/26652). To try it out follow: https://github.com/istio/istio/wiki/Dev-Builds#dev-release-information. Support will be included in the upcoming 1.15 release ETA ~1 month.<br> Multi-arch images starting to appear [here](https://hub.docker.com/r/istio/proxyv2/tags) with 1.15.0-beta releases.<br> 2) [Istio container build instructions](https://github.com/aws/aws-graviton-getting-started/blob/main/containers-workarounds.md#Istio)|
```

Comment:
Can we change the ETA to an actual date?  What do you mean by starting to appear?  Are they available?

Reply:
I don’t know when the release date is, currently 1.15.0 beta images are available, at some point soon the non-beta images will become available. We can wait to update the page until then but I was trying to be as current as possible and indicate support is coming and Beta images are available now.

Reply:
Can say: "We expect the release to be in October 2022", just to avoid needing someone to look at when this page was updated to know when ~1 month is past-due.

Reply:
Sure, it could get released any day now so Sept/Oct is appropriate - I'll update accordingly.